static/js/recent_topics_ui.js

```diff
@@ -870,8 +870,7 @@ export function show() {
     compose_closed_ui.update_buttons_for_recent_topics();
 
     narrow_state.reset_current_filter();
-    const recent_topics_title = $t({defaultMessage: "Recent conversations"});
-    narrow.set_narrow_title(recent_topics_title);
+    narrow.update_narrow_title(narrow_state.filter());
```

Comment:
After this change, does it make sense to stop exporting `set_narrow_title`, since it would no longer be called from outside `narrow.js`?